messages/pt.json

```
@@ -3,5 +3,6 @@
   "store/app-store-components-agree-terms": "Ao fazer este pedido, você concorda com os {appTermsAndConditions}",
   "store/app-store-components-terms-and-conditions": " termos e condições do {appName}.",
   "store/app-store-components-install-page-button": "Vá para a página de instalação",
-  "store/app-store-components-confirmation-message": "Obrigado por sua compra!{newLine}Para completar o processo de instalação na conta {account}, use o botão abaixo"
+  "store/app-store-components-confirmation-message": "Obrigado por sua compra!{newLine}Para completar o processo de instalação na conta {account}, use o botão abaixo",
+  "store/app-store-components-billing-notice-message": "Será cobrada uma taxa de US $ 0,01 para validar as informações do seu cartão de crédito. Isso será reembolsado assim que este pagamento for aprovado"
```

A moeda não está atrelada ao idioma da mensagem, provavelmente teria que receber por parâmetro ou emitir (e possivelmente exibir em outro lugar)

```suggestion
  "store/app-store-components-billing-notice-message": "Será cobrada uma taxa de {currency} 0,01 para validar as informações do seu cartão de crédito. Isso será reembolsado assim que este pagamento for aprovado"
```